integration_test/tests/util.rs

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: CC0-1.0
+
+//! Tests for methods found under the `== Util ==` section of the API docs.
+
+#![allow(non_snake_case)] // Test names intentionally use double underscore.
+#![allow(unused_imports)] // Because of feature gated tests.
+
+use bitcoin::hex::FromHex as _;
+use bitcoin::{Amount};
+use integration_test::{Node, NodeExt as _, Wallet};
+use node::vtype::*; // All the version specific types.
+
+#[test]
+fn utils__createmultisig__modelled() {
+    let node = Node::with_wallet(Wallet::Default, &[]);
+    node.fund_wallet();
+
+    // Use two new addresses to get public keys.
+    let addr1 = node.client.new_address().expect("new_address 1");
+    let addr2 = node.client.new_address().expect("new_address 2");
+    let info1 = node.client.validate_address(&addr1).expect("validateaddress 1");
+    let info2 = node.client.validate_address(&addr2).expect("validateaddress 2");
+
+    let pubkey1 = info1.address.as_ref().unwrap();
+    let pubkey2 = info2.address.as_ref().unwrap();
+
+    // For test purposes, just use the addresses as dummy pubkeys (in real use, use actual pubkeys).
+    let keys = vec![pubkey1.clone(), pubkey2.clone()];
```

Comment:
Once `validateaddress` has strong types in `model` then we can pass `PublicKey`s in here.